commands/inventory/add_agent_qan_mysql_perfschema_agent.go

```diff
@@ -111,4 +133,7 @@ func init() {
 	AddAgentQANMySQLPerfSchemaAgentC.Flag("disable-queryexamples", "Disable collection of query examples").BoolVar(&AddAgentQANMySQLPerfSchemaAgent.DisableQueryExamples)
 	AddAgentQANMySQLPerfSchemaAgentC.Flag("tls", "Use TLS to connect to the database").BoolVar(&AddAgentQANMySQLPerfSchemaAgent.TLS)
 	AddAgentQANMySQLPerfSchemaAgentC.Flag("tls-skip-verify", "Skip TLS certificates validation").BoolVar(&AddAgentQANMySQLPerfSchemaAgent.TLSSkipVerify)
+	AddAgentQANMySQLPerfSchemaAgentC.Flag("tls-ca", "Path to certificate authority certificate file").StringVar(&AddAgentQANMySQLPerfSchemaAgent.TLSCaFile)
```

Comment:
Have you tried `FileVar`? https://pkg.go.dev/gopkg.in/alecthomas/kingpin.v2#FlagClause.FileVar

Reply:
Yes I tried it, but it will return os.File and there is no method to read all so first you need get length of file and then read it from 0 to length, look easier how we doing it now, or at least same.

Reply:
It is not important but you could use `ioutil.ReadAll` since file implements the `Reader` interface. 
We use string everywhere else so it is OK.